l10n_it_withholding_tax_payment/i18n/l10n_it_withholding_tax_payment.pot

```diff
@@ -0,0 +1,268 @@
+msgid "Date Payment"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date_start
+msgid "Date Start"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__date_stop
+msgid "Date Stop"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__display_name
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: selection:withholding.tax.move.payment,state:0
+msgid "Draft"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model_terms:ir.ui.view,arch_db:l10n_it_withholding_tax_payment.wizard_wt_move_payment_create_view
+msgid "Generate"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__id
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__id
+msgid "ID"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__journal_id
+msgid "Journal"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment____last_update
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__write_uid
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__write_uid
+msgid "Last Updated by"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__write_date
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_wizard_wt_move_payment_create__write_date
+msgid "Last Updated on"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move_payment__line_ids
+msgid "Lines"
+msgstr ""
+
+#. module: l10n_it_withholding_tax_payment
+#: model:ir.model.fields,field_description:l10n_it_withholding_tax_payment.field_withholding_tax_move__wt_move_payment_id
+msgid "Move Payment"
```

Comment:
Se è inteso come "Movimento di pagamento" (e non come pulsante che innesca un'azione) allora ci andrebbe "Payment Move".

Reply:
👌

Reply:
Anche qui lascerei la lettera maiuscola come in origine.

Reply:
👌